tests/ui-toml/toml_disallowed_methods/clippy.toml

```diff
@@ -14,4 +14,5 @@ disallowed-methods = [
     "conf_disallowed_methods::Struct::method",
     "conf_disallowed_methods::Trait::provided_method",
     "conf_disallowed_methods::Trait::implemented_method",
+    "conf_disallowed_methods :: path :: with :: spaces",
```

Comment:
Was this requested?

Reply:
Not that I know of. I assumed not allowing spaces was an oversight (but maybe I'm missing something).

Reply:
I see now that the `.split("::")` pattern appears many places, and that a larger fix is required. I will back this change out for now.